packages/ui-components/src/lib/storesGeneric/cachedWritableStore.ts

```
@@ -0,0 +1,71 @@
+		key,
+		defaultValue,
+		(v) => (v ? serialize(v) : ''),
+		(v) => (v ? deserialize(v) : undefined)
+	);
+
+export const cachedWritableIntOptional = (key: string, defaultValue = undefined) =>
+	cachedWritableOptionalStore<number>(
+		key,
+		defaultValue,
+		(v) => v.toString(),
+		(v) => parseInt(v)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)